projects/fuzzing-puzzles/project.yaml

```diff
@@ -10,4 +10,4 @@ sanitizers:
   - undefined
```

Comment:
@jonathanmetzman do you think it's going to work fine, or should I change to ASan?

Reply:
I don't think UBSan will work at all with AFL